server/streamServer.go

```diff
@@ -14,35 +14,43 @@ import (
 )
 
 const (
-	ReadBufferSize  = 1024
-	WriteBufferSize = 1024
-	StreamInterval  = 100 * time.Millisecond
+	readBufferSize        = 1024
+	writeBufferSize       = 1024
+	defaultStreamInterval = 100 * time.Millisecond
 )
 
 type ConnectionType struct {
 	SubscribeAllStateUpdates bool
 }
 
 type StreamServer struct {
-	// lock for protobuf packets
+	// Websocket port
+	Port int
+
+	// Stream interval
+	StreamInterval int
+
+	// Lock for protobuf packets
 	ProtoBufPacketsMutex sync.Mutex
 
-	// list of client message data packets to be sent to client
+	// Socket request router
+	SocketRequestRouter ISocketRequestRouter
+
+	// Client message data packets to be broadcasted
 	ClientEventsQueue []ClientEvent
 
+	// Table updates to be broadcasted
 	TableUpdatesQueue []state.TableUpdate
```

Comment:
Were these updates always part of StreamServer? I like it here but they are going to be read from a stream, I would like to see them have a read write lock

Reply:
Yeah it's always been here - think we're good for now